src/github/webhook.py

```diff
@@ -89,31 +89,32 @@ def _handle_pull_request_review_comment(payload: dict):
     # This is NOT the node_id, but is a numeric string (the databaseId field).
     review_database_id = payload["comment"]["pull_request_review_id"]
 
-    with dynamodb_lock(pull_request_id):
-        if action in ("created", "edited"):
-            pull_request, comment = graphql_client.get_pull_request_and_comment(
-                pull_request_id, comment_id
-            )
-            if not isinstance(comment, PullRequestReviewComment):
-                raise Exception(
-                    f"Unexpected comment type {type(PullRequestReviewComment)} for pull"
-                    " request review"
-                )
-            review: Optional[Review] = Review.from_comment(comment)
-        elif action == "deleted":
-            pull_request = graphql_client.get_pull_request(pull_request_id)
-            review = graphql_client.get_review_for_database_id(
-                pull_request_id, review_database_id
+    if action in ("created", "edited"):
+        pull_request, comment = graphql_client.get_pull_request_and_comment(
+            pull_request_id, comment_id
+        )
+        if not isinstance(comment, PullRequestReviewComment):
+            raise Exception(
+                f"Unexpected comment type {type(PullRequestReviewComment)} for pull"
+                " request review"
             )
-            if review is None:
-                # If we deleted the last comment from a review, Github might have deleted the review.
-                # If so, we should delete the Asana comment.
-                logger.info("No review found in Github. Deleting the Asana comment.")
-                github_controller.delete_comment(comment_id)
-        else:
-            raise ValueError(f"Unexpected action: {action}")
+        review: Optional[Review] = Review.from_comment(comment)
+    elif action == "deleted":
+        pull_request = graphql_client.get_pull_request(pull_request_id)
+        review = graphql_client.get_review_for_database_id(
+            pull_request_id, review_database_id
+        )
+    else:
+        raise ValueError(f"Unexpected action: {action}")
 
+    with dynamodb_lock(pull_request_id):
+        if review is None:
```

Comment:
Does this need to be a separate `if` statement if we have line 116? `if...else` should cover that right?